simexpal/launch/slurm.py

```diff
@@ -138,7 +138,11 @@ def _represents_int(x):
 				'-e', os.path.join(cfg.basedir, 'aux/_slurm/' + log_pattern + '.err')])
 
 		if use_array:
-			sbatch_args.append('--array=0-' + str(len(locked) - 1))
+			if experiment.max_simultaneous:
+				max_simultaneous = f"%{experiment.max_simultaneous}"
+			else:
+				max_simultaneous = ""
+			sbatch_args.append(f"--array=0-{len(locked) - 1}{max_simultaneous}")
```

Comment:
It would be cool to provide these arguments using a python API such as [PySlurm](https://github.com/PySlurm/pyslurm), but I see this is out of scope for this PR.